### ISSUE TITLE
Collapsible categories

### ISSUE THREAD

**Issue by 7eventy7** [7eventy7/Self-Serve#7]
Would it be possible to make the categories collapsible?

**Comment by Beatso** [comment]
I could probably do this, although I wouldn't know how to do it animated. Is this ok?

**Comment by 7eventy7** [comment]
Yeah that should be fine, I have a friend whose gonna try and help me customize the site to be unique

**Comment by Beatso** [comment]
cool
